src/utils/redirectUtils.ts

```diff
@@ -1,6 +1,12 @@
-import { roomId } from "../stores/setup";
+import { Cookies } from "react-cookie";
+import { getRoomId } from "./connectionOptions";
```

Comment:
getRoomId sound like you have "knowlege" about the context, where it should be used. Is this maybe more a contextId, or parentId?